solr/contrib/ltr/src/java/org/apache/solr/ltr/model/LambdaMARTModel.java

```diff
@@ -25,20 +25,78 @@
 import org.apache.lucene.search.Explanation;
 import org.apache.solr.ltr.feature.Feature;
 import org.apache.solr.ltr.norm.Normalizer;
-import org.apache.solr.ltr.util.LTRUtils;
+import org.apache.solr.util.SolrPluginUtils;
 
 public class LambdaMARTModel extends LTRScoringModel {
 
-  private final List<RegressionTree> trees = new ArrayList<RegressionTree>();
+  private final HashMap<String,Integer> fname2index;
```

Comment:
No particular reason (other than it previously being `final HashMap<String,Integer> fname2index = new HashMap<String,Integer>();` local variable) i.e. can change.

Added a commit to reduce the number of `fname2index` references throughout the code (by making the RegressionTree\[Node\] classes (and the methods used to create them) non-static).